lib/src/test/CMakeLists.txt

```diff
@@ -23,17 +35,16 @@ if (JNI_FOUND)
 endif()
 
 include_directories(
-/opt/hostedtoolcache/jdk/11.0.17/x64/include
-/opt/hostedtoolcache/jdk/11.0.17/x64/include/linux
-${CMAKE_SOURCE_DIR}/../../../../../main/cpp
-${CMAKE_SOURCE_DIR}/../../../../../main/cpp/include/libkiwix
-${CMAKE_SOURCE_DIR}/../../../../../main/cpp/include/libzim
-${CMAKE_SOURCE_DIR}/../../../../../main/cpp/include/javah_generated
-${CMAKE_SOURCE_DIR}/../../../../../main/cpp/include/utils
+/opt/hostedtoolcache/jdk/11.0.18/x64/include
+/opt/hostedtoolcache/jdk/11.0.18/x64/include/linux
```

Comment:
We must not have absolute path in CMakeLists.txt file (neither in any build configuration file)